playbooks/files/rax-maas/plugins/conntrack_count.py

```diff
@@ -50,9 +63,50 @@ def get_metrics():
     return metrics
 
 
+def get_metrics_lxc_container(container=''):
+
+    # Return 0 values if we can't determine current state
+    # from a container
+    metrics = {
+        'nf_conntrack_count': {'value': 0},
+        'nf_conntrack_max': {'value': 0}}
+
+    # Create lxc container object
+    container = lxc.Container(container)
+
+    # Check if container is even running
+    try:
+        assert(container.init_pid > 1)
```

Comment:
Could these be explicitly checked via an `if` and raised with some details, rather than `assert`ed? The `status_error` below is just going to end up having a bare `AssertionError` if these fail.

Reply:
I can reformat those not to use assert